crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -353,6 +353,12 @@ define_tedge_config! {
             /// forwarded to Cumulocity on the `s/us` topic
             #[tedge_config(example = "c8y", default(function = "c8y_topic_prefix"))]
             topic_prefix: TopicPrefix,
+
+            mqtt: {
```

Comment:
Since the `bridge` itself is an MQTT concept, I'm wondering if this additional `mqtt` sub-section is redundant.

Reply:
fixed in 45bf3d9a7f0ed6fa76c33dc496cf4dfaeea149f5